spec/features/idv/steps/confirmation_step_spec.rb

```diff
@@ -47,6 +50,24 @@
       acknowledge_and_confirm_personal_key
       expect(page).to have_current_path(account_path)
     end
+
+    context 'with personal key confirmation disabled' do
+      let(:idv_personal_key_confirmation_enabled) { false }
+
+      before do
+        click_continue if javascript_enabled?
+      end
+
+      it 'does not display modal content' do
+        expect(page).not_to have_content t('forms.personal_key.title')
+        expect(page).not_to have_content t('forms.personal_key.instructions')
+      end
+
+      it 'continues to the account page' do
+        expect(current_path).to eq(account_path)
+        expect(page).to have_content t('headings.account.verified_account')
+      end
```

Comment:
These specs are slow and each one will create a new browser session and run through the entire proofing flow. Could we collapse these to a single test case?